frontend/src/lib/components/PropertyFilters/Filter/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
```

Comment:
I think this extra `Filter` folder is confusing. I would instead consider putting the main components in the root of `PropertyFilters` and maybe add all the supporting components in a `components` folder.